src/mem3/src/mem3.erl

```diff
@@ -323,8 +323,8 @@ range(#ordered_shard{range = Range}) ->
     Range;
 range(<<"shards/", Start:8/binary, "-", End:8/binary, "/", _/binary>>) ->
     [
-        httpd_util:hexlist_to_integer(binary_to_list(Start)),
-        httpd_util:hexlist_to_integer(binary_to_list(End))
+        list_to_integer(binary_to_list(Start), 16),
```

Comment:
We could even go a tiny bit faster by using `binary_to_integer/2`

```
./erlperf 'httpd_util:hexlist_to_integer(binary_to_list(<<"abc123def980">>)).' 'binary_to_integer(<<"abc123def980">>, 16).' --samples 10 --warmup 2
Code                                                                       ||        QPS       Time     Rel
binary_to_integer(<<"abc123def980">>, 16).                                  1   28289 Ki      35 ns    100%
httpd_util:hexlist_to_integer(binary_to_list(<<"abc123def980">>)).          1   20880 Ki      47 ns     73%
```

Reply:
Oh cool! Will update.